### ISSUE TITLE
SVG with some pathes are not drawn

### ISSUE THREAD

**Issue by slavslavin** [wieslawsoltes/Svg.Skia#173]
Hi,
I have some SVG files that are not drawn correctly. 
One of them:
```html
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="16px" height="16px" viewBox="0 0 16 16" enable-background="new 0 0 16 16" xml:space="preserve">
  <g id="main">
    <g id="icon">
      <path d="M0,14L0.07852,14.39L0.293,14.71L0.6107,14.92L1,15L14,15L14.39,14.92L14.71,14.71L14.92,14.39L15,14L15,7.77L14.97,7.996L14.9,8.232L14.62,8.632L14.22,8.901L13.98,8.974L13.73,9L12.22,9L11.98,8.975L11.74,8.902L11.35,8.633L11.09,8.232L11,7.735L11,5.584L10.9,5.684L10.54,5.919L10.12,6L10.12,6L9.685,5.912L9.33,5.673L9.09,5.319L9.024,5L0,5zz" fill-rule="evenodd" fill="#E5BD67"/>
      <path d="M0.6107,1.079L0.293,1.293L0.07852,1.611L0,2L0,4L9,4L9,3L8.6,3L8.286,2.969L7.988,2.878L7.712,2.731L7.469,2.531L6.406,1.469L6.162,1.269L5.887,1.122L5.588,1.031L5.274,1L1,1zz" fill-rule="evenodd" fill="#96762C"/>
    </g>
    <g id="overlay">
      <path d="M10.0699,2.7712c-0.0451,0.0478,-0.0701,0.111,-0.0699,0.1767l0.0006,1.9354c0.0003,0.1039,0.1259,0.1557,0.1994,0.0823l1.8,-1.7219V7.735c0,0.1416,0.0833,0.265,0.2249,0.265h0.5601h0.502h0.4391c0.1416,0,0.2563,-0.1148,0.2563,-0.2563V3.2524l1.8185,1.7304c0.0737,0.0732,0.1991,0.0209,0.1991,-0.0829V2.9776c0,-0.0662,-0.0256,-0.1299,-0.0715,-0.1776c0,0,-2.0193,-1.9735,-2.7965,-2.7414c-0.0795,-0.0786,-0.204,-0.078,-0.2828,0.0013C12.0873,0.8265,10.0699,2.7712,10.0699,2.7712z" fill-rule="evenodd" fill="#3E79B4"/>
    </g>
  </g>
</svg>
```

![изображение](https://github.com/wieslawsoltes/Svg.Skia/assets/116277940/428e0e57-ebef-499f-86ef-5f620361fff6)

Мaybe this is related to: #https://github.com/wieslawsoltes/Svg.Skia/issues/135

**Comment by slavslavin** [comment]
The issue occurs when the `<path>` tag value end with "zz". If value ends with one "z" image are drawn correctly.

**Comment by wieslawsoltes** [comment]
Will be fixed in 1.0.2 release
![image](https://github.com/wieslawsoltes/Svg.Skia/assets/2297442/3665f26e-aed9-4cbb-856d-bb008796dae3)
